Sources/Build/BuildDescription/SwiftTargetBuildDescription.swift

```diff
@@ -582,6 +582,17 @@ public final class SwiftTargetBuildDescription {
             args += ["-emit-module-interface-path", self.parseableModuleInterfaceOutputPath.pathString]
         }
 
+        if self.defaultBuildParameters.prepareForIndexing {
+            args += [
+                "-Xfrontend", "-enable-library-evolution",
```

Comment:
Is `-enable-library-evolution` needed here now?

Reply:
Was waiting for an update toolchain. There's one there now. Should be ready to remove.

Reply:
Would you mind updating the diff accordingly before merging?

Reply:
Will do. That will be the last piece. One I can make sure it works 😀

Reply:
If this is the only thing blocking the PR from being merged, I’d prefer to get the PR merged with `-enable-library-evolution` still set because it could unblock me to test preparation in sourcekit-lsp. I know that I’ll get spurious warnings because of library evolution but I can ignore those for now.

Reply:
Yeah. As mentioned above, I'm not even sure if the change that was made even fixes it. I have one more fix Ben B mentioned about hiding the flag and then I'll kick off the tests.